pkgs/top-level/all-packages.nix

```diff
@@ -8232,6 +8232,13 @@ in
   gcc-arm-embedded-9 = callPackage ../development/compilers/gcc-arm-embedded/9 {};
   gcc-arm-embedded = gcc-arm-embedded-9;
 
+  gcc-arm-bin-9 = { target } : callPackage ../development/compilers/gcc-arm-bin/9 { target = target; };
```

Comment:
What is the difference between the precompiled ones and `pkgsCross.armv7l-hf-multiplatform.buildPackages.gcc`?

Reply:
The precompiled ones are sort of "golden reference" versions provided by Arm (regarding exact compiler version, flags etc.). Some projects (e.g. OP-TEE) would download them, but I'm more comfortable with keeping such stuff in the nix store instead of copies cluttering my home.